src/game.cpp

```diff
@@ -3173,6 +3188,53 @@ void game::debug()
         break;
 
         case 18: {
+            uilist wind_direction_menu;
+            wind_direction_menu.text = _( "Select new wind direction:" );
+            wind_direction_menu.addentry( 0, true, MENU_AUTOASSIGN, wind_direction_override == -1 ?
+                                          _( "Keep normal wind direction" ) : _( "Disable direction forcing" ) );
+            int count = 1;
+            std::vector<int> angles { 0, 45, 90, 135, 180, 225, 270, 315 };
```

Comment:
Why this vector? You could simply increment by 45 degrees in the following loop.